src/use-swr.ts

```diff
@@ -442,7 +452,8 @@ function useSWR<Data = any, Error = any>(
         // we have to ignore the revalidation result (res) because it's no longer fresh.
         // meanwhile, a new revalidation should be triggered when the mutation ends.
         if (
-          MUTATION_TS[key] &&
+          // Since ts start at 0
+          MUTATION_TS[key] !== undefined &&
```

Comment:
I think ts starts from 1, now returns `++ts`

Reply:
https://github.com/vercel/swr/blob/eb7fa36920fa5eaca5d68a61ece6eabe54c64907/src/use-swr.ts#L110 😂

Reply:
how about bump init ts from 0 to 1? then the `now() - 1` will start from 1. I guess it's safe to have it since ts is for comparsion?

Reply:
I personally feels that ```MUTATION_TS[key] !== undefined``` might be batter here. 
Because in here we want to check the existence of the mutation timestamp. 
But I don't have strong opinion here.

Reply:
Another problem here is that once `MUTATION_TS` is assigned, it will never get deleted (or reset to 0/undefined). So I think that 2 `mutate()` calls will fail here?